bin/emqx

```diff
@@ -343,7 +349,7 @@ case "$1" in
                 HEART_OPTION="start_boot"
                 ;;
         esac
-        RUN_PARAM="$@"
```

Comment:
why is this change ?

Reply:
By SC2124.
https://github.com/koalaman/shellcheck/wiki/SC2124

> If you want to assign N elements as 1 element by concatenating them, use * instead of @, e.g. myVar=${myArray[*]} (this separates elements with the first character of IFS, usually space).